src/isambard/evaluation/packing_density.py

```diff
@@ -0,0 +1,45 @@
+"""Module for evaluating the packing density of a polymer or assembly. """
+
+import ampal
+import numpy as np
+
+
+def calculate_packing_density(structure, radius=7):
```

Comment:
I'd recommend changing the name to `tag_packing_density` to match other function of this type.